the-association/About-us.rst

```diff
@@ -0,0 +1,71 @@
+The Odoo Community Association, or OCA, is a nonprofit organization whose
+mission is to support the collaborative development of the Odoo features and to
+promote the widespread use of Odoo. The Association provides financial,
+organizational and legal support to the broader Odoo Open Source community.
+It also serves as an independent legal entity to which community members can
+contribute to the code, the funding and other resources, with the knowledge
+that their contributions will be maintained for public benefit.
+
+**The Association’s projects are all freely available and usable under an
+OSI-certified Open Source license.**
+
+=========
+Our Goals
+=========
+
+* Help and **promote** the collaborative software development of Odoo;
+* Encourage the development of Odoo and its features while **coordinating** and
+  organizing the collaborative work on the software;
+* Assist the community while **defending its interests** and the sustainability
+  of its developments;
+* **Promote** the use of the Odoo solution;
+* Facilitate synergies, collaborations and **fund raising** efforts;
+* Actively **collaborate** on the definition of the roadmaps of new versions of
+  the tool and their implementation.
+
+=================
```

Comment:
Should this be top level?

Reply:
They are all at the same level on the original page: 
https://odoo-community.org/page/About